src/View.php

```diff
@@ -347,11 +347,10 @@ public function renderFile(string $viewFile, array $parameters = [], ?ViewContex
             $viewFile = $this->theme->applyTo($viewFile);
         }
 
-        if (is_file($viewFile)) {
-            $viewFile = $this->localize($viewFile);
-        } else {
+        if (!is_file($viewFile)) {
             throw new ViewNotFoundException("The view file does not exist: $viewFile");
         }
+        $viewFile = $this->localize($viewFile, $parameters['language'] ?? null, $parameters['sourceLanguage'] ?? null);
```

Comment:
`language` and `sourceLanguage` are quite common names likely to be used in view. Therefore, it will cause trouble because `language` will be often passeed in the context of the view parameter to be used in the view itself.

Reply:
@vjik suggested solution is to pass it in `ViewContextInterface`.